.github/workflows/build_on_push.yaml

```diff
@@ -29,11 +33,22 @@ jobs:
           config: .markdownlint.json
           globs: '**/*.md'
 
+      - name: YAML Lint
+        uses: ibiqlik/action-yamllint@v3
+        with:
+          config_file: .yamllint.yaml
+          file_or_dir: |
+            .github/workflows/
```

Comment:
Is there a reason to limit checking to specific files? I think I'd like all files checked.